index.html

```diff
@@ -1,75 +1,114 @@
-<!doctype html>
+<!DOCTYPE html>
 
 <html lang="en">
-<head>
-  <meta charset="utf-8">
+  <head>
+    <meta charset="utf-8" />
 
-  <title>Sprint Challenge - Home</title>
+    <title>Sprint Challenge - Home</title>
 
-  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
-
-</head>
-
-<body>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto|Rubik"
+      rel="stylesheet"
+    />
+    <!-- <a style = "text-decoration:none"></a> -->
+    <link rel="stylesheet" href="css/index.css" />
+  </head>
+  <!-- -------------------Ignore above --------------------------------------------- -->
+  <body>
+    <header>
+      <div class="lambdaLogo">
+        <h1><img src="img/lambda-black.png" alt="Lambda" /></h1>
+      </div class = "header">
+      <nav>
+        <a href="#"><h2>Home</h2></a>
+        <a href="/about.html"><h2>About</h2></a>
+        <a href="#"><h2>Products</h2></a>
+        <a href="#"><h2>Blog</h2></a>
+        <a href="#"><h2>Contact</h2> </a>
+      </nav>
```

Comment:
When you are finished with your project, try to remove any commented-out code to keep it clean and organized.